docs/web-apps/automated-testing/_partials/_advanced-cypress.md

```diff
@@ -24,4 +24,12 @@ You can pass env var `grep` to filter tests by keyword.
 
 ```bash
 saucectl run --env grep=".type()"
+```
+
+## Filtering Cypress w/ Cucumber Tests
```

Comment:
No need for slang here :D Also better SEO if we use the words `tags` in the title (most likely what the user will be searching for).

```suggestion
## Filtering Cypress Cucumber Tests via Tags
```